python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_partition_backfill.py

```diff
@@ -242,11 +251,33 @@ def _execute_backfill_iteration_with_side_effects(graphql_context, backfill_id):
     """Executes an asset backfill iteration with side effects (i.e. updates run status and bulk action status)."""
     with get_workspace_process_context(graphql_context.instance) as context:
         backfill = graphql_context.instance.get_backfill(backfill_id)
-        list(
-            execute_asset_backfill_iteration(
-                backfill, logging.getLogger("fake_logger"), context, graphql_context.instance
+        evaluation_time = pendulum.now("UTC")
+        tick = graphql_context.instance.create_tick(
+            TickData(
+                instigator_origin_id=_BACKFILL_ORIGIN_ID,
+                instigator_name=backfill_id,
+                instigator_type=InstigatorType.BACKFILL,
+                status=TickStatus.STARTED,
+                timestamp=evaluation_time.timestamp(),
+                selector_id=backfill_id,
```

Comment:
`selector_id` is only used for querying when there is an instigators table, but table doesn't always exist. my plan was to use the backfill_id as the selector_id so that we can get just the ticks for a specific backfill in the UI, but if the instigators table doesn't exist, the query will fetch all ticks for all backfills. should i set the backfill as the `instigator_origin_id` instead?